src/libstore/meson.build

```diff
@@ -99,6 +100,20 @@ deps_public += nlohmann_json
 sqlite = dependency('sqlite3', 'sqlite', version : '>=3.6.19')
 deps_private += sqlite
 
+# AWS C++ SDK has bad pkg-config
+aws_s3 = dependency('aws-cpp-sdk-s3', required : false)
+configdata.set('ENABLE_S3', aws_s3.found().to_int())
+if aws_s3.found()
+  aws_s3 = declare_dependency(
+    include_directories: include_directories(aws_s3.get_variable('includedir')),
+    link_args: [
+      '-L' + aws_s3.get_variable('libdir'),
+      '-laws-cpp-sdk-s3',
```

Comment:
The makefile has `-laws-cpp-sdk-transfer -laws-cpp-sdk-s3 -laws-cpp-sdk-core -laws-crt-cpp`, aren't those needed here (maybe for static linking)?

Reply:
Oh thanks you!

I guess I first did this on the other store settings branch, with its missing symbols, and didn't get as a far as the link error.

Reply:
Fixed now.